### DISCUSSION TITLE
Serve localhost over a (named)pipe

### DISCUSSION THREAD

**Discussion by Commaster** [bcb/jsonrpcserver#231] [Q&A]
Greetings!

Since we are defaulting to localhost for the serve() function is it possible to skip the http/tcp overhead (and binding a port) and use local pipes or sockets instead?
(I think they are called namedpipes on windows)

Thank you

**Comment by bcb** [comment]
The `serve` function uses Python’s built-in http server but you can use any protocol or framework by using `dispatch` instead of `serve`. Check the docs.
